search/result.hpp

```diff
@@ -153,33 +163,34 @@ class Results
 
   bool AddResult(Result && result);
 
-  // Fast version of AddResult() that doesn't do any duplicates checks.
+  // Fast version of AddResult() that doesn't do any duplicates
```

Comment:
s/duplicates checks/checks for duplicates/

Reply:
Done